apps/gatsby/src/AppConfig/ContentTypesPanel.js

```diff
@@ -52,16 +52,18 @@ export const ContentTypesList = ({
   }
 
   return contentTypes.map(({ sys, name }) => (
-    <CheckboxField
-      key={sys.id}
-      labelIsLight
-      labelText={name}
-      name={name}
-      checked={enabledContentTypes.includes(sys.id)}
-      value={sys.id}
-      onChange={() => onContentTypeToggle(sys.id)}
-      id={sys.id}
-    />
+    <div>
```

Comment:
Why the wrapping div here? Sounds looks a hacky fix?

Reply:
No actually to achieve a line break. The current visuals looked not good with the new forma updates and the spacing was totally off because all the content types were on the same line.

Reply:
Okay then maybe the new version changed the `display` type...